fairscale/utils/testing.py

```diff
@@ -303,7 +304,14 @@ def replacement(*args: Any, **kwargs: Any) -> None:
 
             error_queue = multiprocessing.get_context("spawn").SimpleQueue()
             if "OMPI_COMM_WORLD_RANK" in os.environ:
+                # TODO (Min): this global used to be assigned every time this file is imported.
+                # I changed it to be assigned on first use. Should be the same, but I am not
+                # sure this is used or is correct since different processes would have different
```

Comment:
Can you clarify what the TODO action item here is?

Reply:
I'd love to. However, I don't have the full background on this function. I think it was introduced for mpi based test with pipe. Maybe @msbaines can chime in? I only wanted to make sure we don't leave behind a lot of /tmp files running a test. For a test with world_size==2, this creates 3 files for a single test case since the master process and 2 worker processes each create a file due to this.